packages/cli/src/databases/migrations/sqlite/1693491613982-ExecutionSoftDelete.ts

```diff
@@ -0,0 +1,5 @@
+import { ExecutionSoftDelete1693491613982 as BaseMigration } from '../common/1693491613982-ExecutionSoftDelete';
+
+export class ExecutionSoftDelete1693491613982 extends BaseMigration {
+	transaction = false as const;
```

Comment:
Always found this option name `transaction` rather unclear... I wonder if others do as well. Maybe we could rename to `disableForeignKeys` and flip the condition downstream?

Reply:
yeah. we have to use this flag unfortunately to disable typeorm's internal transactions, so that we can run the `PRAGMA` commands and then wrap the migration in our own transaction.
The proper way to "fix" this would be to fork typeorm, and automatically disable foreign keys constraints is there was another flag like `disableForeignKeys` present on the migration.